src/rich-text-editor.component.js

```diff
@@ -190,9 +191,17 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
   }
 
   function setHTML(html) {
-    emptyEditor()
-    divRef.current.innerHTML = richTextContext.sanitizeHTML(html, 'setHTML')
-    focus()
+    if(!initialMount.current) {
+      // both emptyEditor() and focus() result in unwanted autofocus on contentEditable on first render
+      // if the consumer wants autofocus, give us the prop
+      emptyEditor()
+      divRef.current.innerHTML = richTextContext.sanitizeHTML(html, 'setHTML')
+      focus()
+    } else {
+      initialMount.current = false
+      divRef.current.innerHTML = richTextContext.sanitizeHTML(html, 'setHTML')
+      if (props.autofocus) {focus()}
```

Comment:
if you're going to go for a same-liner if statement, might as well go all the way and exclude the curly braces 😄 

```suggestion
      if (props.autofocus) focus()
```